src/content/components/Router/Router.js

```diff
@@ -480,15 +483,26 @@ export class Router extends React.PureComponent {
         },
       },
       { spacer: true },
+      {
+        label: "Settings",
+        icon: "settings",
+        routeProps: {
+          path: "/settings",
+          component: SettingsView,
+        },
+        devOnly: true,
+      },
       {
         label: "About Bugzy",
         icon: "info",
         routeProps: {
           path: "/about",
-          component: Preferences,
+          component: AboutView,
         },
       },
-    ];
+    ].filter(
+      router => !router.devOnly || process.env.NODE_ENV !== "production"
```

Comment:
Filtering is done around line 115. You can just add a new condition in the filter function up there for this, to return null. No reason to actually exclude this route for production, just exclude the list item. That way we can still use it by entering the URL directly

Reply:
And better to avoid looping over all of these twice